runtime/stdlib/crypto.go

```diff
@@ -98,9 +98,9 @@ func NewCryptoContract(
 	return compositeValue, nil
 }
 
-func cryptoAlgorithmEnumConstructorType(
+func cryptoAlgorithmEnumConstructorType[T sema.CryptoAlgorithm](
```

Comment:
Needed because Go slices are invariant

Reply:
@dreamsmasher This isn't necessarily bad, haha (https://stackoverflow.com/questions/6684493/why-are-arrays-invariant-but-lists-covariant)